client/src/templates/Introduction/components/super-block-accordion.css

```diff
@@ -350,3 +350,14 @@ button .block-header-button-text {
     margin-top: -10px;
   }
 }
+
+@media screen and (max-width: 500px) {
+  .super-block-accordion .block-label {
+    margin-inline-start: auto;
+    margin-inline-end: 10px;
+  }
+  
```

Comment:
Sorry, my suggestion contained space characters on this line, which Prettier didn't like.

```suggestion

```